src/frontend/lib/file-system.ts

```diff
@@ -0,0 +1,3 @@
+import {ExternalDirectoryPath} from 'react-native-fs';
+
+export {ExternalDirectoryPath as EXTERNAL_FILES_DIR};
```

Comment:
Same variable used by Mapeo Mobile, just to confirm: https://github.com/digidem/mapeo-mobile/blob/56528dcccf291d4248beb6348767ca50a6ce92be/src/frontend/api.ts#L420